lib/logstash/filters/mutate.rb

```diff
@@ -220,17 +220,18 @@ def filter(event)
 
   private
   def remove(event)
-    # TODO(sissel): use event.sprintf on the field names?
     @remove.each do |field|
+	  field = event.sprintf(field)
       event.remove(field)
     end
   end # def remove
 
   private
   def rename(event)
-    # TODO(sissel): use event.sprintf on the field names?
     @rename.each do |old, new|
+	  old = event.sprintf(old)
       next unless event.include?(old)
+	  new = event.sprintf(new)
```

Comment:
new is a reserved work in ruby, you should change it.  Did you try running this? I believe this would not run because of this error.